understanding/22/target-size-minimum.html

```diff
@@ -88,9 +88,14 @@ <h3>Spacing</h3>
                 <figcaption>For a square/rectangular target, the 24 CSS pixel diameter circle is centered on the target itself. For convex and concave targets, it is centered on the bounding box of the shape. Note the concave target, where in this case the center of the bounding box is outside of the actual target (image shown to scale - <a href="img/target-size-bounding-boxes.svg">see the scalable original version</a>)</figcaption>
             </figure>
 
-            <p>We repeat this process for all adjacent undersized targets. To determine if an undersized target has sufficient spacing (to pass this Success Criterion's spacing exception), we check that the 24 CSS pixel diameter circle of the target does not intersect another target or the circle of any other adjacent undersized targets.</p>
+            <p>We repeat this process for all adjacent undersized targets. To determine if an undersized target has sufficient spacing (to pass this Success Criterion's spacing exception), we check that the 24 CSS pixel diameter circle of the target does not intersect another target, or the circle of any other adjacent undersized targets.</p>
```

Comment:
Disagree with removing this comma; it's grammatically correct for it to be omitted because the clause after the "or" isn't a complete sentence.

```suggestion
            <p>We repeat this process for all adjacent undersized targets. To determine if an undersized target has sufficient spacing (to pass this Success Criterion's spacing exception), we check that the 24 CSS pixel diameter circle of the target does not intersect another target or the circle of any other adjacent undersized targets.</p>
```